lib/node_modules/@stdlib/array/base/slice/docs/types/index.d.ts

```diff
@@ -211,28 +57,16 @@ declare function slice( x: Complex128Array, start: number, end: number ): Comple
 * @returns output array
 *
 * @example
-* var Complex64Array = require( '@stdlib/array/complex64' );
-*
-* var x = new Complex64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+* var x = [ 1, 2, 3 ];
 *
 * var out = slice( x, 0, 3 );
-* // returns <Complex64Array>[ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ]
-*/
-declare function slice( x: Complex64Array, start: number, end: number ): Complex64Array;
-
-/**
-* Returns a shallow copy of a portion of an array.
-*
-* @param x - input array
-* @param start - starting index (inclusive)
-* @param end - ending index (exclusive)
-* @returns output array
+* // returns [ 1, 2, 3 ]
 *
 * @example
 * var x = [ 1, 2, 3, 4, 5, 6 ];
 *
-* var out = slice( x, 0, 3 );
-* // returns [ 1, 2, 3 ]
+* var out = slice( x, 0, 2 );
+* // returns [ 1, 2 ]
```

Comment:
```suggestion
* var out = slice( x, 0, 3 );
* // returns [ 1, 2, 3 ]
```

It is not clear why this change was ever made.